src/components/Unavaliable/styles.ts

```diff
@@ -0,0 +1,40 @@
+import {createStyles, rem} from "@mantine/core";
+
+export const useStyles = createStyles((theme) => ({
+    root: {
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: 'center'
+    },
+
+    label: {
+        textAlign: 'center',
+        fontWeight: 900,
+        fontSize: rem(220),
+        lineHeight: 1,
+        marginBottom: `calc(${theme.spacing.xl} * 1.5)`,
```

Comment:
if it is possible should get rid of difficult logic in css, maybe think about layout in common sense